ops/terraform/services/base/values/prod-sbx.yaml

```
@@ -145,7 +145,7 @@
 /bfd/${env}/server/nonsensitive/db/wrapper/plugins_csv: "initialConnection,auroraConnectionTracker,readWriteSplitting,efm2,failover"
 /bfd/${env}/server/nonsensitive/db/wrapper/host_selection_strategy: roundRobin
 /bfd/${env}/server/nonsensitive/db/wrapper/init_connection_strategy: roundRobin
-/bfd/${env}/server/nonsensitive/pac/enabled: "true"
+/bfd/${env}/server/nonsensitive/pac/enabled: "false"
```

I think we want to leave these on outside of prod. There's no risk to sensitive data here.
```suggestion
/bfd/${env}/server/nonsensitive/pac/enabled: "false"
```